bpf/Makefile

```diff
@@ -96,6 +95,9 @@ endif
 endif
 
 MAX_LB_OPTIONS = $(MAX_BASE_OPTIONS) -DENABLE_NAT46=1 -DENCAP_IFINDEX=1
+ifneq ("$(KERNEL)","49")
+MAX_LB_OPTIONS += -DLB_SELECTION=1 -DLB_SELECTION_MAGLEV=1
```

Comment:
The `bpf/Makefile: Disable Maglev for bpf_host and bpf_lxc` shouldn't be backported to v1.8 as Maglev was introduced in v1.9.